README.md

```diff
@@ -20,37 +34,32 @@ at the conclusion of the tests by default.
 
 Apply the resource claim as follows to re-create
 the namespace, Prometheus, Grafana and dependencies for further
```

Comment:
why is the claim 're-creating' the ns, Prometheus, and Grafana?

Reply:
How about we offer a make cluster so that the workflow does not duplicate the installation after Uptest cleaned up its test setup? Users who want to run make e2e could do it purposefully as a separate step with or without pre-existing cluster.

Reply:
Added a make cluster option.